pkg/cli/jobtemplate/describe.go

```diff
@@ -66,15 +67,28 @@ func DescribeJobTemplate(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		for _, jobTemplate := range jobTemplates.Items {
+		for i, jobTemplate := range jobTemplates.Items {
+			// Remove managedFields
+			jobTemplate.ManagedFields = nil
 			PrintJobTemplateDetail(&jobTemplate, describeJobTemplateFlags.Format)
+			// Print a separator if it's not the last element
+			if len(jobTemplates.Items) != 1 && i < len(jobTemplates.Items)-1 {
+				fmt.Println("---------------------------------")
+			}
```

Comment:
Can you show the output?

Reply:
root@VM-0-8-ubuntu:/home/ubuntu/volcano/_output/bin# vcctl jobtemplate describe
apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: a
  namespace: default
  resourceVersion: "1882"
  uid: 2162e4b5-c540-4bcc-bcad-05f8c6ddff9b
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: b
  namespace: default
  resourceVersion: "1883"
  uid: 210895b2-86bc-434e-abf1-014d82c127b5
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: c
  namespace: default
  resourceVersion: "1884"
  uid: 48e51823-533f-404a-acdf-37a630e6b2e2
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: d
  namespace: default
  resourceVersion: "1885"
  uid: 0c7ec7f4-22b3-4049-9225-4f93226d6dee
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: e
  namespace: default
  resourceVersion: "1886"
  uid: 72c9bb25-591a-4533-a7e7-1f90e5734ec1
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}

Reply:
```bash
root@VM-0-8-ubuntu:/home/ubuntu/volcano/_output/bin# vcctl jobtemplate describe
apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: a
  namespace: default
  resourceVersion: "1882"
  uid: 2162e4b5-c540-4bcc-bcad-05f8c6ddff9b
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: b
  namespace: default
  resourceVersion: "1883"
  uid: 210895b2-86bc-434e-abf1-014d82c127b5
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: c
  namespace: default
  resourceVersion: "1884"
  uid: 48e51823-533f-404a-acdf-37a630e6b2e2
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: d
  namespace: default
  resourceVersion: "1885"
  uid: 0c7ec7f4-22b3-4049-9225-4f93226d6dee
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}


apiVersion: flow.volcano.sh/v1alpha1
kind: JobTemplate
metadata:
  creationTimestamp: "2024-08-27T11:09:53Z"
  generation: 1
  name: e
  namespace: default
  resourceVersion: "1886"
  uid: 72c9bb25-591a-4533-a7e7-1f90e5734ec1
spec:
  maxRetry: 5
  minAvailable: 1
  plugins:
    env: []
    ssh: []
    svc: []
  policies:
  - action: RestartJob
    event: PodEvicted
  priorityClassName: high-priority
  queue: default
  schedulerName: volcano
  tasks:
  - name: default-nginx
    replicas: 1
    template:
      metadata:
        creationTimestamp: null
        name: web
      spec:
        containers:
        - command:
          - sh
          - -c
          - sleep 10s
          image: nginx:1.14.2
          imagePullPolicy: IfNotPresent
          name: nginx
          resources:
            requests:
              cpu: "1"
        restartPolicy: OnFailure
status: {}
```

Reply:
I changed to use spaces instead of separators